### ISSUE TITLE
Visualize Route Deviations

### ISSUE THREAD

**Issue by greghutch** [googlemaps/fleet-debugger#61]
Add a toggle button that will show all the generated routes for a vehicle.   Ideally compute a 'deviation score' based on actual route driven vs initial guided route.

**Comment by regeter** [comment]
Agreed, this is a pain point when trying to investigate routing issues. The generated route should be on the trip event, but unclear if this is available in cloud logging. Will investigate.

**Comment by regeter** [comment]
I've added Polyline UI as a quick workaround.
With Cloud Logging, I am afraid it does not include the projected route on the trips, maybe we could enable that.
